plugins/drng/plugin.go

```diff
@@ -83,13 +52,17 @@ func configureEvents() {
 				log.Info(err)
 				return
 			}
-			if err := Instance.Dispatch(msg.IssuerPublicKey(), msg.IssuingTime(), parsedPayload); err != nil {
+			if err := instance.Dispatch(msg.IssuerPublicKey(), msg.IssuingTime(), parsedPayload); err != nil {
 				//TODO: handle error
 				log.Info(err)
 				return
 			}
-			log.Info(Instance.State.Randomness())
+			log.Info(instance.State.Randomness())
 		})
 	}))
+}
 
+// Enabled returns the enabled status of the plugin
+func Enabled() bool {
+	return Plugin.Status != node.Disabled
```

Comment:
This check is not correct as it does not check for dynamically disabled plugins.
Remove this function and use `node.IsSkipped` from `github.com/iotaledger/hive.go/node` where needed.